### DISCUSSION TITLE
STATE CODE ERROR

### DISCUSSION THREAD

**Discussion by pisanialessandro** [themustafaomar/jsvectormap#64] [General]
Hi!
There is an error with FR code.
The French_Guiana is GF - https://it.wikipedia.org/wiki/ISO_3166-2:GF not france!

![image](https://user-images.githubusercontent.com/17181327/152126741-5dacc1a2-4631-4d48-9555-f83092e82ad1.png)

**Comment by themustafaomar** [comment]
Hi @pisanialessandro 

You're actually right, but I've nothing to do here since this map is generated via [jvectormap converter](https://jvectormap.com/documentation/gis-converter/) depending on https://www.naturalearthdata.com/downloads/.

Editing the SVG manually is really complicated because both France and French Guiana share the same path element.

However, there may be another version of the shape that has the two areas splitted, I'm not sure about that.

Of course, a PR is welcome.
